pyccel/codegen/printing/cwrappercode.py

```diff
@@ -425,23 +464,17 @@ def _body_management(self, used_names, variable, collect_var, cast_function, che
         Responsible for calling functions that take care of body creation
         """
```

Comment:
Could you add `Parameters` to this docstring please.

Are all the arguments still used?

Reply:
true I have set the castfunction to None and forgot to remove it

Reply:
done 15bdea2